src/DrCabal/Interactive.hs

```diff
@@ -0,0 +1,85 @@
+{-# LANGUAGE NumericUnderscores #-}
+
+{- |
+Module                  : DrCabal.Interactive
+Copyright               : (c) 2019 Alexander Gugel
+                          (c) 2022 Dmitrii Kovanikov
```

Comment:
I have no complaints on the current copyright, you can even put your name here if you want 🙂 

But I'm curious about the original code 🤔 Where a similar code was already written?

Reply:
My bad, I just copied header from `DrCabal.Watch`.